scripts/r/04_refit/refit_jvc.Rmd

```diff
@@ -531,7 +531,7 @@ Currently, the GitHub repo "many_speech" is not available. We have added a zippe
 
 unzip(here("data", "analyses", "jvc", "cromileptes_saxatilis", "many_speech.zip"), exdir = here("data", "analyses", "jvc", "cromileptes_saxatilis"))
 
-cromileptes_saxatilis_durs <- read_csv(here("data", "analyses", "jvc", "cromileptes_saxatilis", "many_speech", "processed_data", "NF_durations.csv")) %>% 
+cromileptes_saxatilis_durs <- read_csv(here("data", "analyses", "jvc", "cromileptes_saxatilis", "processed_data", "NF_durations.csv")) %>% 
```

Comment:
In the revised script the files are unzipped to a many_speech folder. Pls, change back to original.

Reply:
The code will not run with that because the unzipped file doesn't add this directory. Try and run it.

Reply:
Ah ha. There error I was getting was from the zip file not being downloaded. Not sure why, but the API for OSF is hit or miss. Reverting back.